### PR TITLE
Removes Bluespace Crystals and Diamonds from emergency roller bed's recipe

### PR DESCRIPTION
**Pull request by dis-integrates-the-integration-tests**
## About The Pull Request

This PR _should_ remove the 0.5 sheets of bluespace crystals and diamonds required in the "Medical Bed (Emergency)" recipe.

## Why It's Good For The Game

The roller bed just seems like an ordinary item, it IS an ordinary item, and functions like an ordinary item. I see no reason that it would require bluespace crystals, since it doesn't use any sort of bluespace wizardry. SURE it would make sense.. if you could fit 5 patients in one roller bed and then fold it then fit it in your bag, but right now? Yeah, it doesn't. Besides: It's already locked behind Research, and bluespace crystals are already a valuable enough resource.

and then we have the diamonds. Why does a **roller bed** require diamonds? Sure, titanium makes sense, for the integrity of the roller bed, but... Diamonds? It's already a scarce enough material that runs out pretty quickly, and it's not so much better than wheelchairs (5 Iron)

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>

image
image
image

</details>

## Changelog
:cl: KannaLisvern
qol: Removed Bluespace Crystals and Diamonds from medical emergency bed's recipe.
/:cl:

### CODE REVIEW

File: code/modules/research/designs/medical_designs.dm [hunk 1]
```diff
@@ -352,7 +352,7 @@
 	desc = "A portable, foldable version of the medical bed. Perfect for paramedics or whenever you have mass casualties!"
 	id = "medicalbed_emergency"
 	build_type = PROTOLATHE | AWAY_LATHE
-	materials = list(/datum/material/titanium = SHEET_MATERIAL_AMOUNT * 2.7, /datum/material/plastic = SHEET_MATERIAL_AMOUNT * 1.7, /datum/material/diamond = SMALL_MATERIAL_AMOUNT * 5, /datum/material/bluespace = SMALL_MATERIAL_AMOUNT * 5)
+	materials = list(/datum/material/titanium = SHEET_MATERIAL_AMOUNT * 2.7, /datum/material/plastic = SHEET_MATERIAL_AMOUNT * 1.7, /datum/material/diamond = SMALL_MATERIAL_AMOUNT * 5)
```
Comment (Fikou) [inline code/modules/research/designs/medical_designs.dm:355]: why dos it require diamonds

Comment (Fikou) [reply code/modules/research/designs/medical_designs.dm:355]: this isnt very much more effective than a bodybag which you have like 20 of roundstart or like a rolling chair

**Comment by lessthnthree** [comment]
I agree with the removal of diamonds/bluespace, but the price should be slightly higher than the non-upgraded version of the bed, this change makes it equal.

**Comment by Fikou** [comment]
> I agree with the removal of diamonds/bluespace, but the price should be slightly higher than the non-upgraded version of the bed, this change makes it equal.
> 
> The emergency variant of the bed was previously not available to be built by the player, you were limited to however many you had on the map.

whats the non upgraded version of the bed? this item has no subtypes

**Comment by ExcessiveUseOfCobblestone** [comment]
I really like that theres really no reason for this to be titanium/plastic either, since the medical bed (what it shares a cost with) doesn't do anything that a normal bed cant do, specifically they share the same surgery timer. It can fold ig but im hard pressed to believe that's worth titanium and plastic.

**Comment by FlufflesTheDog** [comment]
These are pretty much a straight downgrade to using a body bag (which fit in boxes, can hold multiple people/bodies and any dropped belongings) except for being more roleplay-friendly, which makes it feel weird being any sort of expensive. But maybe that's a problem on body bags rather than the beds. 
Either way they should probably also be removed from the miniaturized bluespace research if they're no longer made with bluespace crystals.

**Comment by lessthnthree** [comment]
> > I agree with the removal of diamonds/bluespace, but the price should be slightly higher than the non-upgraded version of the bed, this change makes it equal.
> > The emergency variant of the bed was previously not available to be built by the player, you were limited to however many you had on the map.
> 
> whats the non upgraded version of the bed? this item has no subtypes

/obj/structure/bed/medical
/datum/design/medical_bed

**Comment by ExcessiveUseOfCobblestone** [comment]
is this done or do u want to hit it harder

**Comment by ExcessiveUseOfCobblestone** [comment]
idk just seems too expensive ill probably let someone hit it more later if they wanted lol (the price, nothing else)

**Pull request merged**